scripts/biomedical/Human-GEM_Human1/README.md

```diff
@@ -0,0 +1,153 @@
+# Importing Human-GEM Human1 data
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+    1. [Download URL](#download-url)
+    2. [Database Overview](#database-overview)
+    3. [Schema Overview](#schema-overview)
+    4. [Notes and Caveats](#notes-and-caveats)
+    5. [License](#license)
+2. [About the Import](#about-the-import)
+    1. [Scripts](#scripts)
+    2. [tMCF files](#tMCF-files)
+    3. [Example](#example)
+    
+    
+## About The Dataset
+
+### Download URL
+
+The input tsv files, which were generated by extracting data from [Human-Gem.xml](https://github.com/SysBioChalmers/Human-GEM/blob/master/model/Human-GEM.xml) file in [Human-GEM repository](https://github.com/SysBioChalmers/Human-GEM), were obtained from [this repository](https://github.com/paul-shannon/bio-gdb/tree/main/forSam/v1). <br>
```

Comment:
Maybe this is a question for Sam... it sounds like there is more code to convert from the official data source.  Do we plan to check those scripts in?  For example, in case the person's github account above goes away and we need to regenerate from source...

Reply:
@spiekos